src/dbnode/namespace/types.go

```diff
@@ -213,7 +213,7 @@ type SchemaRegistry interface {
 
 // Metadata represents namespace metadata information
 type Metadata interface {
-	// Equal returns true if the provide value is equal to this one
+	// Equal returns true if the provided value is equal to this one
```

Comment:
nit; mind adding `.` to all of these including the existing ones?

Reply:
👍